test/unit/test_conn.c

```diff
@@ -37,7 +38,7 @@ static void connCloseCb(struct conn *conn)
 	FIXTURE_RAFT;     \
 	FIXTURE_CLIENT;   \
 	struct conn conn; \
-	bool closed;
+	uint64_t closed;
```

Comment:
why not just make this field `queue *`?

Reply:
Overall solution with bool closed is "hackerish". I'd prefer not to make it even more convoluted.